lib/Dialect/TritonGPU/Transforms/HoistTMEMAlloc.cpp

```diff
@@ -92,6 +92,56 @@ class RemoveUnusedTMEMLoad : public OpRewritePattern<TMEMTokenLoadOp> {
   }
 };
 
+std::optional<bool> getBoolFromConstant(Value cst) {
+  auto constantOp = cst.getDefiningOp<arith::ConstantOp>();
+  if (!constantOp) {
+    return std::nullopt;
+  }
+  assert(constantOp.getValue());
+  if (auto boolAttr = dyn_cast<BoolAttr>(constantOp.getValue())) {
+    return boolAttr.getValue();
+  }
+  return std::nullopt;
+}
```

Comment:
Make it a utility given https://github.com/triton-lang/triton/blob/0f1e09e308fa71544dd833f768305425c9f2c383/lib/Dialect/TritonGPU/Transforms/OptimizeAccumulatorInit.cpp#L174-L184?

Reply:
Done